tasks/configure.js

```diff
@@ -10,8 +10,15 @@ var fetchres = require('fetchres');
 
 function fetchFromNextConfigVars(source, target, key) {
 	console.log(`Fetching ${source} config from Next Config Vars for ${target}`);
-	return fetch('https://ft-next-config-vars.herokuapp.com/production/' + source, { headers: { Authorization: key } })
+	const configEnv = process.env.CONFIG_ENV || 'production';
```

Comment:
Thoughts on making this into a CLI option? Would make it a self-documenting feature then, and you might be able to validate that the value is valid too.